src/bp-activity/bp-activity-filters.php

```diff
@@ -80,7 +80,6 @@
 add_filter( 'bp_acomment_name', 'stripslashes_deep', 5 );
 add_filter( 'bp_get_activity_action', 'stripslashes_deep', 5 );
 add_filter( 'bp_get_activity_content', 'stripslashes_deep', 5 );
-add_filter( 'bp_get_activity_content_body', 'stripslashes_deep', 5 );
```

Comment:
@janakkaneriya007 I don't think we need to remove this filter as I post the content into the activity `wp-content\themes\` then it showing the correct see here https://prnt.sc/wcvsb9 maybe something wrong with the discussion content.